### PR TITLE
Expand negative contractions in DOS5 content

### PR DESCRIPTION
**Pull request by lfdebrux**
Negative contractions (words ending in -n't) are not considered GDS style, so we should remove them.

- haven't -> have not
- shouldn't -> should not
- isn't -> is not
- won't -> will not
- don't -> do not
- doesn't -> does not
- can't -> cannot

### CODE REVIEW

**Review by NoraGDS — APPROVED** [review]

**Review by NoraGDS — APPROVED** [review]

File: frameworks/digital-outcomes-and-specialists-5/questions/declaration/safeguardPersonalData.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Safeguarding personal data
 question: >
   Do you agree to protect personal data according to the government guidance on <a href="https://www.gov.uk/data-protection-your-business" target="_blank" rel="noopener noreferrer">data protection and your business (opens in new tab)</a>?
 type: boolean
-hint: If you provide access to user research studios, for example, please confirm that you don’t share research videos with third parties.
+hint: If you provide access to user research studios, for example, please confirm that you do not share research videos with third parties.
```
Comment (NoraGDS) [inline frameworks/digital-outcomes-and-specialists-5/questions/declaration/safeguardPersonalData.yml:5]:
```suggestion
hint: If you provide access to user research studios, for example, can you confirm that you do not share research videos with third parties.
```

**Review by NoraGDS — APPROVED** [review]

**Review by NoraGDS — APPROVED** [review]

**Review by NoraGDS — APPROVED** [review]

File: frameworks/digital-outcomes-and-specialists-5/questions/declaration/safeguardPersonalData.yml (hunk 1 above)
Comment (NoraGDS) [reply frameworks/digital-outcomes-and-specialists-5/questions/declaration/safeguardPersonalData.yml:5]: Does this change to remove 'please' still make sense with the response options?

Comment (lfdebrux) [reply frameworks/digital-outcomes-and-specialists-5/questions/declaration/safeguardPersonalData.yml:5]: Let's leave these changes to a separate PR

**Review by NoraGDS — CHANGES_REQUESTED** [review]
Comment on last change about removing 'please'

**Review by NoraGDS — APPROVED** [review]

**Pull request merged**